src/App.js

```diff
@@ -1,19 +1,144 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import TodoList from './components/TodoList/TodoList';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
 
+import todos from './api/todos';
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-}
+const preparedTodos = todos.map((item) => {
+  const user = users.find(human => human.id === item.userId);
+  const todo = {
+    user,
+    ...item,
+  };
+
+  return todo;
+});
+
+export class App extends Component {
+  state = {
+    TODOs: preparedTodos,
```

Comment:
```suggestion
    TODOs: preparedTodos,
```
почему большие буквы?

Reply:
Скопировал вставил из ридми, потом отвлёкся. Каюсь!!!